web/js/map-OpenLayers.js

```diff
@@ -749,6 +749,16 @@ $.extend(fixmystreet.utils, {
             fixmystreet.map.setCenter(centre, fixmystreet.zoom || 3);
         }
 
+        // map.getCenter() returns a position in "map units", but sometimes you
+        // want the center in GPS-style latitude/longitude coordinates (WGS84)
+        // for example, to pass as GET params to fixmystreet.com/report/new.
+        fixmystreet.map.getCenterWGS84 = function() {
+            return fixmystreet.map.getCenter().transform(
```

Comment:
Could this cut decimals to 6dp, pointless having more.

Reply:
We use `.transform()` a few times in `map-OpenLayers.js`, but I don’t think we actively round to 6 decimal places in any of those situations – why start now?

Reply:
I was thinking about it only recently, in fact; those other places should presumably do it too. Just because they don't yet doesn't mean this one shouldn't, though :)

Reply:
Yes, comment poorly worded on my part! I meant more "why change it here and not everywhere else" ;-)

If it’s a think we’re going to keep doing everywhere, do you think we should just create our own wrapper around `.transform()` that does the rounding, and use that everywhere instead?

Reply:
There's only one other obvious use I can see that would benefit from it, in `update_pin` (and the admin drag callback is already doing it), so probably not worth it.